extensions/blocks/eventbrite/save.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { RichText, getColorClassName } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { createWidgetId, eventIdFromUrl } from './utils';
+
+/**
+ * Adapted button save function from @wordpress/block-library
+ * (Using Gutenberg code that shipped with WordPress 5.3)
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/wp/5.3/packages/block-library/src/button/save.js
+ *
+ * Uses a "button" element rather than "a", since the button opens a modal rather than a link.
+ */
+
+function saveButton( eventId, attributes ) {
+	const {
+		backgroundColor,
+		borderRadius,
+		customBackgroundColor,
+		customTextColor,
+		text,
+		textColor,
+	} = attributes;
+
+	const textClass = getColorClassName( 'color', textColor );
+	const backgroundClass = getColorClassName( 'background-color', backgroundColor );
+
+	const buttonClasses = classnames( 'wp-block-button__link', {
+		'has-text-color': textColor || customTextColor,
+		[ textClass ]: textClass,
+		'has-background': backgroundColor || customBackgroundColor,
+		[ backgroundClass ]: backgroundClass,
+		'no-border-radius': borderRadius === 0,
+	} );
+
+	const buttonStyle = {
+		backgroundColor: backgroundClass ? undefined : customBackgroundColor,
+		color: textClass ? undefined : customTextColor,
+		borderRadius: borderRadius ? borderRadius + 'px' : undefined,
+	};
+
+	return (
+		<div>
+			<RichText.Content
```

Comment:
This is copied directly from how Gutenberg saves the Button block, but using `<button>` instead of `<a>`, since it's an on page action in this context, rather than a link.

It's possible the `<Button>` component could also work, but I think that component is intended for rending a button in the admin view, whereas `<RichText>` seems specifically for rendering the saved block markup that's output in the page content.